vignettes/adsl.Rmd

```diff
@@ -271,7 +271,7 @@ This call would return the input dataset with the column `DCSREAS` and `DCSREASP
 By default, the function will map 
 
 - `DCSREAS` as `DSDECOD` if `DSDECOD` is not `"COMPLETED"` or `NA`, `NA` otherwise
-- `DCSREASP` as `DSTERM` if `DSDECOD` is not `"COMPLETED"` or `NA`, `NA` otherwise
+- `DCSREASP` as `DSTERM` if `DSDECOD` is not `"COMPLETED"` or `NA`, and `DSDECOD` is equal to "OTHER",`NA` otherwise
```

Comment:
A space is missing after the second comma.

I would remove the first comma.

Please use backticks for `"OTHER"`.